client/src/components/chat-header/chat-header.component.jsx

```diff
@@ -17,39 +31,81 @@ const ChatHeader = () => {
     }
   }, [currentlyActive]);
 
+  const triggerSidebar = () => {
+    dispatch({
+      type: DirectoryActionTypes.TOGGLE_SIDEBAR,
+    });
+  };
+
   return (
-    <Box boxShadow={1} py={3} px={5}>
-      <Grid container direction="row" justify="space-between" alignItems="center">
-        <Grid item>
-          <Grid container direction="row" justify="flex-start" alignItems="center" spacing={2}>
-            <Grid item>
-              <Typography>{title}</Typography>
+    <Box className={classes.header}>
+      <Box className={classes.bigPadd}>
```

Comment:
a box in a box, can we add these 2 boxes inheriting both of those class properties together?

Reply:
fixed in d095ea3